src/components/Docs/SnippetViewer/CheckRequestViewer.tsx

```diff
@@ -88,34 +81,44 @@ const { allowed } = await fgaClient.check({
       /* eslint-disable no-tabs */
       return `
 options := ClientCheckOptions{
-\tAuthorizationModelId: openfga.PtrString("${modelId}"),
+    AuthorizationModelId: "${modelId}",
 }
+
 body := ClientCheckRequest{
-\tUser:     "${user}",
-\tRelation: "${relation}",
-\tObject:   "${object}",${
-        !contextualTuples
-          ? ''
-          : `
-\tContextualTuples: &[]ClientTupleKey{
+    User:     "${user}",
+    Relation: "${relation}",
+    Object:   "${object}",${
+      !contextualTuples
+        ? ''
+        : `
+    ContextualTuples: []ClientTupleKey{
 ${
   !contextualTuples
     ? ''
     : contextualTuples
         .map(
-          (tuple) => `\t\t{
-\t\t\tUser:     "${tuple.user}",
-\t\t\tRelation: "${tuple.relation}",
-\t\t\tObject:   "${tuple.object}",
-\t\t}`,
+          (tuple) =>
+            `        {
+            User:     "${tuple.user}",
+            Relation: "${tuple.relation}",
+            Object:   "${tuple.object}",
+        },`,
         )
-        .join(',\n')
+        .join('\n')
 }
-\t}`
-      }
+    },`
+    }${
+      context
+        ? `
+    Context: &map[string]interface{}${JSON.stringify(context)},`
```

Comment:
does this work? 🤔

Reply:
JSON is not a native go representation AFAIK

Reply:
I guess it works for primitive types, but as soon as you have non-primitives it doesn't. Not sure the best way to marshal an arbitrary structure in native Go format when doing so in Javascript. You have any ideas that aren't ridiculously cumbersome to maintain?

Reply:
Is there code to convert from arbitrary json to a struct?

Reply:
Is there code to convert from json to a Go struct that is written in Javascript? I don't know of anything. That's the challenge.